models/Recipe.model.js

```diff
@@ -2,9 +2,15 @@ const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
```

Comment:
Te marca undefined por que tu modelo esta vacio 👀 
Esta es una propuesta, hay especificaciones que puedes hacer en tus modelos, desde el enum que es un array con los valores permitidos hasta el default que es el valor por defecto en caso de que no lo proporcionemos 
```suggestion

const recipeSchema = new Schema({
    title: {type: String, required: true, unique: true},
    level: {type: String, enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"]},
    ingredients: {type: Array},
    cuisine: {type: String, required: true},
    dishType: {type: String, enum: ["breakfast", "main_course", "soup", "snack", "drink", "dessert", "other"]},
    image: {type: String, default: "https://images.media-allrecipes.com/images/75131.jpg"},
    duration: {type: Number, min: 0},
    creator: {type: String},
    created: {type: Date, default: Date.now }
});```